### DISCUSSION TITLE
PSA- Atom Lite and ESPresense Firmware

### DISCUSSION THREAD

**Discussion by TCoughlin02** [ESPresense/ESPresense#451] [General]
TLDR: Use generic esp32 firmware on this device, atom-matrix firmware was unreliable.

I have six of these in my home for room tracking and was having all sorts of connectivity issues with devices disconnecting every few minutes.  Except for one.  After taking a look a that device I noticed it was on the latest generic firmware while the other 5 were on the Atom-matrix firmware.  After re flashing devices last night, I have had little to no disconnections.

**Comment by iCSpotRun** [comment]
Oddly, I attempted multiple flashes with the atom matrix firmware before I observed stability. Also, I find when using the generic firmware, the LED doesn't work.

Atoms Lites are becoming popular. I hope the developers can investigate these issues soon.

**Reply by TCoughlin02** [reply]
The LED isn't an issue for me, as I turned them off in the options anyways.

I really like the Atom lites.  The small form factor is great, and the built in case puts it over the top if using it for a stand alone ESPresense device in my opinion.
